cmake/common.cmake

```diff
@@ -88,6 +88,10 @@ option(SAN "Enable Address and Undefined Behavior Sanitizers" OFF)
 option(DISABLE_QUOTE_VERIFICATION "Disable quote verification" OFF)
 option(BUILD_END_TO_END_TESTS "Build end to end tests" ON)
 option(COVERAGE "Enable coverage mapping" OFF)
+option(AZURE_LOG_ANALYTICS "Enable azure analytics log collection" OFF)
```

Comment:
I think we can remove this option, now that we use an env var instead?

Reply:
This has actually already been removed, I think GitHub is slow in updating today